pkg/client/unversioned/request.go

```diff
@@ -48,6 +48,16 @@ import (
 // are therefore not allowed to set manually.
 var specialParams = sets.NewString("timeout")
 
+// backoff is the underlying backoff struct we use to prevent thundering herd drama, esp when this runs in a controller.
+var backoff = &URLBackoff{Backoff: util.NewBackOff(1*time.Second, 120*time.Second)}
```

Comment:
Please remove this global. Note that this is probably not appropriate even as a client variable, because the server may limit different endpoints at different rates.

E.g., suppose a list is getting 429'd, and a watch (for a different resource) happens at the same time and is accepted. You don't want the watch to clobber the list's backoff.

We already have a mechanism for preventing clients from sending too much traffic to a server, that is the client rate-limit. Please *don't* add a second way of fixing that problem when making client retries back off, just make sure that each retry kicks the rate limiter first.